src/loaders/jobs/taskfarmer/taskfarmer_task_mgr.py

```diff
@@ -0,0 +1,357 @@
+import datetime
```

Comment:
I'm guessing the next step after this PR is figuring out what's already been run and resubmitting just failures / incompletes?

Reply:
yup. that sounds like a good next step.

Reply:
:+1: